godtools/App/Services/Renderer/Views/MobileContent/Views/ContentText/MobileContentTextViewModel.swift

```diff
@@ -77,16 +79,34 @@ class MobileContentTextViewModel: MobileContentTextViewModelType {
     }
     
     var textAlignment: NSTextAlignment {
-               
-        let modelTextAlignment: MobileContentTextAlignment? = textModel.textAlignment ?? containerModel?.textAlignment
+        
+        let textModelAlignment = textModel.textAlignment
+        let containerModelAlignment = containerModel?.textAlignment
+        
+        //first determine the desired alignment for left-to-right languages
+        var modelTextAlignment = textModelAlignment ?? containerModelAlignment
+        
+        //if primary language is right-to-left, flip the alignment
+        if languageTextAlignment == .right {
```

Comment:
Hey @reldredge71 this ```languageTextAlignment``` is actually the alignment based on the language direction for the current rendered language.  So if we are viewing the parallel language, then this alignment is based off of that parallel language direction.  It's not related to the primary language in anyway making the above comment not accurate and maybe not what is intended here.

Reply:
@levieggertcru I think I'm following.  I've been forgetting about rendering the parallel language.  So my desired effect would be to first assess what the XML objects state the alignment should be, and then take into account the language currently being rendered and flip the alignment if it is a right-to-left language.  So if the XML says left-aligned and the primary language is left-to-right, then all primary language text should stay left-aligned; if the parallel language is right-to-left, then all parallel language text should be right-aligned.  Is my logic still good, and I need to update the comment?  Or is it my logic that needs to be changed?

Reply:
I can ask, but I think it should flip based on how someone would expect to read the text.  I imagine that a reader of a right-to-left language would expect to read things right-aligned in most cases.  I can still ask though.

Reply:
https://jira.cru.org/browse/GT-1381

It looks like they would like the alignment to be based on the currently rendered langauge

Reply:
Thanks @reldredge71, yeah that makes more sense like you said.